ajax/libs/FormChimp/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "FormChimp",
+  "filename": "jquery.formchimp.min.js",
+  "description": "A customizable MailChimp ajax plugin for jQuery",
+  "author": {
+    "name": "Fabio Quarantini",
+    "url": "http://www.fabioquarantini.com",
+    "twitter": "@febba"
+  },
+  "homepage": "http://www.fabioquarantini.com/formchimp/",
+  "keywords": [
+    "FormChimp",
+    "MailChimp",
+    "jQuery plugin",
+    "ajax"
```

Comment:
Let's add `form` and `newsletter` like https://github.com/fabioquarantini/formchimp/blob/1.2.1/bower.json#L15 and https://github.com/fabioquarantini/formchimp/blob/1.2.1/bower.json#L17